PLUGIN_DIR is defined as ${LIB_DIR}/lmms (/usr/lib64/lmms) by the build system
and is also added to the search path, since its a symlink to /usr/lib we end up
with the same directory but with a different path, resulting in duplicated
plugins.

--- a/src/core/PluginFactory.cpp
+++ b/src/core/PluginFactory.cpp
@@ -63,7 +63,7 @@
 	//      e.g. binary at "C:/Program Files/LMMS/lmms.exe"
 	//           plugins at "C:/Program Files/LMMS/plugins/"
 
-#ifndef LMMS_BUILD_WIN32
+#if !defined(LMMS_BUILD_WIN32) && !defined(PLUGIN_DIR)
 	addRelativeIfExists("../lib/lmms"); // Installed
 #endif
 	addRelativeIfExists("plugins"); // Portable
